### ISSUE TITLE
Module not found

### ISSUE THREAD

**Issue by gayatri-p** [SwapnilSoni1999/spotify-dl#23]
Running on Android 10.0 termux shows this error.
```
internal/modules/cjs/loader.js:798
    throw err;
    ^

Error: Cannot find module 'm3u8stream/lib/parse-time'
Require stack:
- /data/data/com.termux/files/usr/lib/node_modules/spotify-dl/node_modules/ytdl-core/lib/info-extras.js
- /data/data/com.termux/files/usr/lib/node_modules/spotify-dl/node_modules/ytdl-core/lib/info.js
- /data/data/com.termux/files/usr/lib/node_modules/spotify-dl/node_modules/ytdl-core/lib/index.js
- /data/data/com.termux/files/usr/lib/node_modules/spotify-dl/lib/downloader.js
- /data/data/com.termux/files/usr/lib/node_modules/spotify-dl/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:795:15)
    at Function.Module._load (internal/modules/cjs/loader.js:688:27)
    at Module.require (internal/modules/cjs/loader.js:850:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/data/data/com.termux/files/usr/lib/node_modules/spotify-dl/node_modules/ytdl-core/lib/info-extras.js:5:21)
    at Module._compile (internal/modules/cjs/loader.js:958:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:994:10)
    at Module.load (internal/modules/cjs/loader.js:813:32)
    at Function.Module._load (internal/modules/cjs/loader.js:725:14)
    at Module.require (internal/modules/cjs/loader.js:850:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/data/data/com.termux/files/usr/lib/node_modules/spotify-dl/node_modules/ytdl-core/lib/info-extras.js',
    '/data/data/com.termux/files/usr/lib/node_modules/spotify-dl/node_modules/ytdl-core/lib/info.js',
    '/data/data/com.termux/files/usr/lib/node_modules/spotify-dl/node_modules/ytdl-core/lib/index.js',
    '/data/data/com.termux/files/usr/lib/node_modules/spotify-dl/lib/downloader.js',
    '/data/data/com.termux/files/usr/lib/node_modules/spotify-dl/cli.js'
  ]
}
```

PS: ffmpeg installed and running

**Comment by EsmailELBoBDev2** [comment]
same on debian 10

**Comment by nagualcode** [comment]
same on osx

**Comment by EsmailELBoBDev2** [comment]
we need one more to confirm it on apple to proof it not works on all platforms

**Comment by TsPuujee** [comment]
same on mac

**Comment by EsmailELBoBDev2** [comment]
oh wait osx = iphone ? i just noticed it now lol

**Comment by EsmailELBoBDev2** [comment]
@SwapnilSoni1999 so do you have plan ?

**Comment by SwapnilSoni1999** [comment]
I have to inspect which lib causing this gonna update in few days

**Comment by SwapnilSoni1999** [comment]
seems like issue with [https://github.com/fent/node-ytdl-core/pull/520](ytdl-core)
will implement a temporary fix

**Comment by SwapnilSoni1999** [comment]
seems like issue with [ytdl-core](https://github.com/fent/node-ytdl-core/pull/520)
will implement a temporary fix

**Comment by SwapnilSoni1999** [comment]
I guess its still buggy and it cannot fetch new stuff now I've checked the PR on ytdl-core 
I hope they soon pull it and fix

**Comment by SwapnilSoni1999** [comment]
fixed with release 0.1.8
https://github.com/SwapnilSoni1999/spotify-dl/releases/tag/0.1.8
